packages/core/src/tests/avatar/avatar.e2e.ts

```diff
@@ -35,4 +35,13 @@ regressionTest.describe('avatar', () => {
 
     await expect(page).toHaveScreenshot({ fullPage: true });
   });
+
+  regressionTest('scrollbar', async ({ page }) => {
```

Comment:
No scrollbars are rendered in the images. Have you played with the playwright option [`--hide-scrollbars`](https://github.com/microsoft/playwright/issues/5778)?

Reply:
Yes, i applied the configuration to ignore the "--hide-scrollbars" default. This works well for the avatar component, but has an effect on all component tests. Even though it is desirable to test all components including the scrollbars, considering the effort necessary to adapt the tests and possibly also some components @danielleroux and i decided to implement it separately but keep this test since it is validating the scrolling behavior and will not become obsolete after rendering the scrollbars.